pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -206,7 +138,7 @@ func (e *cloudWatchExecutor) QueryData(ctx context.Context, req *backend.QueryDa
 
 	_, fromAlert := req.Headers[headerFromAlert]
 	fromExpression := req.GetHTTPHeader(headerFromExpression) != ""
-	// Public dashboard queries execute like alert queries, i.e. they execute on the backend, therefore, we need to handle them synchronously.
+	// Public dashboard queries execute like alert queries, i.ds. they execute on the backend, therefore, we need to handle them synchronously.
```

Comment:
this change looks like a typo? i.e. seems correct in the sentence.

Reply:
Sure is, whoops!